src/src.builds

```diff
@@ -1,6 +1,10 @@
 <Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
 
+  <PropertyGroup Condition="'$(DotNetBuildFromSource)' == 'true'">
+    <AdditionalBuildConfigurations>$(AdditionalBuildConfigurations);netstandard-$(ConfigurationGroup)</AdditionalBuildConfigurations>
```

Comment:
I thought we already do this but I guess that was only for the refs.

Reply:
Yeah, we need it for the refs so that we can build libs that target netstandard.  Normally we don't need it for the libs for netstandard as there isn't such a thing as a runnable netstandard layout so having a "complete" set of netstandard libs doesn't have a scenario for our normal builds.  Here it is handy because it is a shortcut for additional libs which we know we can build in source-build that will help flesh out the source build packages.